### ISSUE TITLE
api to access to raw bgp update

### ISSUE THREAD

**Issue by SergeKrier** [CAIDA/libbgpstream#204]
What's the C proper api to get access to the raw bgp update from the various sources/brokers ?
can't find in the doc

thanks

**Comment by digizeph** [comment]
C API available at https://bgpstream.caida.org/docs/api/libbgpstream/bgpstream.h.html
Tutorial at https://bgpstream.caida.org/docs/tutorials/libbgpstream

**Comment by SergeKrier** [comment]
i've be through those doc already, by raw, I meant bgp update in binary format

**Comment by SergeKrier** [comment]
seems not available, will try to add

**Comment by alistairking** [comment]
Yeah, it's not currently available, and in fact it's not guaranteed that all data formats supported by BGPStream will even include the raw BGP update information (e.g., we're kinda "lucky" that the RIPE JSON format includes the raw message).

That said, I think it would make sense to expose this information in the record structure in the cases where the data format includes it.

Thanks for offering to try adding this!
